classes/renderer/hasmany.php

```diff
@@ -287,11 +287,22 @@ protected static function getFieldSet($config, $item)
      *
      * @return array
      */
-    protected static function getConfig($item, $data)
+    protected static function getConfig($item, $data, &$renderer_options)
     {
         $class       = get_class($item);
         $config_file = \Config::configFile($class);
         $config      = \Config::load(implode('::', $config_file), true);
+
+        // fallback old
+        if (\Arr::get($renderer_options, 'order', false) && empty(\Arr::get($renderer_options, 'order_property', false))) {
```

Comment:
This option (`order`) currently does not exist, you should update the docs too (but isn't it redundant with `order_property` ?)